.github/workflows/update-server-side-sdk-schema.yml

```diff
@@ -108,23 +123,32 @@ jobs:
         env:
           CI: true
 
-      - name: 'Create branch'
-        run: |
-          git checkout -b feat/open-api-${{ inputs.tag }}
-
       - name: 'Install dependencies'
         run: pnpm install
         working-directory: dx-team-toolkit
 
+      - name: Set git config
+        run: |
+          git config --global --add --bool push.autoSetupRemote true
+
       - name: Update schema
-        run: node dx-team-toolkit/scripts/update-schema.mjs ${{ inputs.schema-path }} ${{ inputs.examples-path }} ${{ inputs.generate-command }} ${{ inputs.tag }} ${{ secrets.GITHUB_TOKEN }}
+        run: node dx-team-toolkit/scripts/update-schema/update-schema.mjs "${{ inputs.schema-path }}" "${{ inputs.examples-path }}" "${{ inputs.generate-command }}" ${{ inputs.tag }} ${{ inputs.appId != '' && steps.app-token.outputs.token || secrets.GH_TOKEN }}
```

Comment:
It is a complicated part of the distribution of this script. May be we can create an action?

Reply:
Refactored script to GitHub action